contracts/injectorFactoryV2.sol

```diff
@@ -53,4 +71,26 @@ contract ChildChainGaugeInjectorV2Factory {
     function getDeployedInjectors() external view returns (address[] memory) {
         return deployedInjectors;
     }
+
+    /**
+     * @notice Registers an injector in the Chainlink Keeper Registry
+     * @param _injector The address of the injector to be registered
+     */
+    function _registerUpkeep(address _injector, address _owner) internal returns (uint256 upkeepId_) {
+        IKeeperRegistrar.RegistrationParams memory registrationParams = IKeeperRegistrar.RegistrationParams({
+            name: "ChildChainGaugeInjectorV2",
+            encryptedEmail: "",
+            upkeepContract: _injector,
+            gasLimit: 2_000_000,
```

Comment:
This gas limit is quite high, at least on mainnet.  Can result in a lot of LINK needing to be paid in but not much being used.  Maybe this should be a configurable parameter.  At best would be good to understand how much gas limit you needed per gauge to injection in 1 go and allow an input like "simultaneous gauges" to define the gas limit.